h/static/scripts/tests/controllers/lozenge-controller-test.js

```diff
@@ -1,65 +1,32 @@
 'use strict';
 
 const LozengeController = require('../../controllers/lozenge-controller');
+const lozengeTemplate = require('./lozenge-template');
+const { setupComponent } = require('./util');
 
 describe('LozengeController', () => {
```

Comment:
Now that LozengeController behaves like other Controller subclasses, the tests can be greatly simplified.